sci-libs/gvar/gvar-13.0.2.ebuild

```diff
@@ -0,0 +1,20 @@
+EAPI=8
+
+DISTUTILS_EXT=1
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 pypi
+
+DESCRIPTION="Gaussian random variables."
+HOMEPAGE="https://github.com/gplepage/gvar"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
```

Comment:
The tarball includes some unittests, can we add support for that in the ebuild?

```
distutils_enable_tests unittest
```

Reply:
tests passed.